txscript/sign.go

```diff
@@ -174,7 +129,8 @@ func signMultiSig(tx *wire.MsgTx, idx int, subScript []byte, hashType SigHashTyp
 		if err != nil {
 			continue
 		}
-		sig, err := RawTxInSignature(tx, idx, subScript, hashType, key)
+		sig, err := RawTxInSignature(tx, idx, subScript, hashType, key,
+			dcrec.STEcdsaSecp256k1)
```

Comment:
This would produce an incorrect signature for an address that is not the standard secp256k1.